### DISCUSSION TITLE
"built with tree-sitter" list

### DISCUSSION THREAD

**Discussion by BrianHicks** [tree-sitter/tree-sitter#1431] [Libraries & tools]
Hey! I asked something like this question in the conference Slack after @dcreager's talk at Strange Loop. Basically: what do y'all think about having a list of tools and projects that are built with tree-sitter? The recent nvim integration springs to mind, but there's also my own [tree-grepper](https://github.com/BrianHicks/tree-grepper).

**Comment by ahelwer** [comment]
It would be neat to have such a list. I also know of cursorless, the vs code plugin for programming via dictation: https://github.com/pokey/cursorless-vscode

edit: seems there is a PR with this same idea: https://github.com/tree-sitter/tree-sitter/pull/1166

**Comment by ahlinc** [comment]
@BrianHicks awesome idea!!! :tada:
May be such page would be great to manage in a wiki tab right in this repo so it would be community editable.

I also had some thoughts about info that would fit into a wiki format:
* Make a community editable page in accompany to [the parsers page](https://tree-sitter.github.io/tree-sitter/#available-parsers) where language authors would be able to leave additional comments and info about their implemented grammars. This can be in a table format with some set of common categories, like:
  * Is a grammar complete or not.
  * Is it fully covered with tests.
  * A link for a language spec used for grammar implementation if it's available.
  * etc. #1094
* Another thought was to create and maintain a page about all language bindings, their completeness and a table with a map of all API function / method names. This would allow to track bindings state and synchronize as much as possible all API function names (there is some discrepancy for now). Also I thought to leverage tree-sitter itself for managing the mapping of actual names in some semi automated way.

/cc: @maxbrunsfeld What do you think if we activate the wiki tab in this repo and create several pages there?

**Comment by calicoday** [comment]
Good ideas. I've also been thinking about bindings info -- I'm working on ruby bindings and I didn't realize at first that many of the calls necessary for query testing are in rust not C. And the javascript calls are slightly different again. Some maps could be very helpful.
